src/test/fuzz/fuzz.cpp

```diff
@@ -92,13 +97,22 @@ void initialize()
             if (std::get<2>(t.second)) continue;
             out_stream << t.first << std::endl;
         }
-        should_abort = true;
+        should_exit= true;
+    }
+    if (should_exit){
+        std::exit(EXIT_SUCCESS);
+    }
+    if (const auto* env_fuzz{std::getenv("FUZZ")}) {
+        // To allow for easier fuzz executable binary modification,
+        static std::string g_copy{env_fuzz}; // create copy to avoid compiler optimizations, and
+        g_fuzz_target = g_copy.c_str();      // strip string after the first null-char.
+    } else {
+        std::cerr << "Must select fuzz target with the FUZZ env var." << std::endl;
```

Comment:
nit: Would it be good to hint at `PRINT_ALL_FUZZ_TARGETS_AND_ABORT=1` in the message?